app/models/application_draft.rb

```diff
@@ -66,7 +66,7 @@ def role_for(user)
   end
 
   def ready?
-    false # valid?(:apply)
+    valid?(:apply)
```

Comment:
We can't merge this PR yet as that would enable the submit button in app/views/application_drafts/new.html.slim:63 but we do not have the `ApplicationDrafts#apply` controller action yet.